components/sidebar/SidebarElement.vue

```diff
@@ -3,6 +3,7 @@ const links: Record<"title" | "url", string>[] = [
   { title: "Home", url: "/" },
   { title: "Browse", url: "/browse" },
   { title: "Search", url: "/search" },
+  { title: "Album", url: "/album/91530" },
```

Comment:
This should be changed once we have a page for viewing all albums

Reply:
Could be resolved by #97